libs/designer/src/lib/core/utils/__test__/outputs.spec.ts

```diff
@@ -66,15 +67,25 @@ describe('Outputs Utilities', () => {
           },
         },
       };
-      const splitOn = "@triggerOutputs()?['body']";
```

Comment:
Can you check if just @triggerBody() in splitON works?

Reply:
Instead of removing the code that was pushing 'body' when the name of the string was 'triggerBody', I added a condition to check if the dereferences already contain 'body' at the first index so it avoids converting triggerBody()?['body'] to body.body for example.

`      // prevents duplicate 'body' property when parsing OpenApi manifests
      if (equals(name, 'triggerBody') && dereferences.length && !equals((dereferences[0].expression as ExpressionLiteral).value, 'body')) {
        properties.push('body');
      }`